src/main/java/com/redhat/exhort/integration/providers/ossindex/OssIndexIntegration.java

```diff
@@ -82,6 +85,32 @@ public void configure() {
         .onFallback()
           .process(responseHandler::processResponseError);
 
+    from(direct("ossIndexHealthCheck"))
+      .routeId("ossIndexHealthCheck")
+      .setProperty(Constants.PROVIDER_NAME, constant(Constants.OSS_INDEX_PROVIDER))
+      .process(excludeOrIncludeProvider)
+      .choice()
+        .when(exchangeProperty(Constants.EXCLUDE_FROM_READINESS_CHECK).isEqualTo(false))
```

Comment:
You can just check if the enabled Services contain the one being tested. If not, you can add the PROVIDER_DISABLED flag in the `healthCheckProviderDisabled` route

```suggestion
        .when(method(vulnerabilityProvider, "getEnabled").contains(Constants.OSV_NVD_PROVIDER))
```